File: hotel-app/src/main/java/eu/getsoftware/hotelico/service/booking/application/customer/common/dto/UserDTO.java
// path: hotelico/service/booking/application/customer/common/dto/UserDTO.java

package eu.getsoftware.hotelico.service.booking.application.customer.common.dto;

import eu.getsoftware.hotelico.clients.common.domain.ids.CustomerDomainEntityId;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * For serialization to JSON!
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserDTO {

    private CustomerDomainEntityId domainEntityId;

    private String title;

    private String firstName;

    private String lastName;

    private String email;

    private String company;

    private String jobTitle;

    // private List<String> languages;

    // private boolean authLinkedIn;

    private String hotelDomainId;
}

File: hotel-app/src/main/kotlin/eu/getsoftware/hotelico/service/booking/application/customer/common/dto/UserDTO.kt
package eu.getsoftware.hotelico.service.booking.application.customer.common.dto;

import eu.getsoftware.hotelico.clients.common.domain.domainIDs.CustomerDomainEntityId

/**
 * for serialisation to JSON!
 */
data class UserDTO (
   
//    var id: Long = 0

    var domainEntityId: CustomerDomainEntityId,

    var title: String? = null,

    var firstName: String? = null,

    var lastName: String? = null,

    var email: String? = null,

    var company: String? = null,

    var jobTitle: String? = null,

//    List<String> languages;

//    boolean authLinkedIn;

    var hotelDomainId: String? = null
) { 
    
}
